src/index.html

```diff
@@ -11,7 +11,124 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" class="form">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+        Surname: <input type="text" name="Surname:">
+        </div>
```

Comment:
Add proper labels
Fix formatting
Fix indentations
```suggestion
        <div class="form__section_field">
          <label for="surname">Surname: </label>

          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
          >
        </div>
```

Apply this pattern to all other fields
Remember to add spaces between sibling elements for better readability

Reply:
Okey, I make fixes please to check again :)